ui/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
```

Comment:
I'd rather add this base info to the root md files so we can localize and standardize expectations there such as adding a section for UI to the developer file